pkgs/applications/window-managers/dwm/dwm-status.nix

```diff
@@ -1,29 +1,25 @@
 { stdenv, lib, rustPlatform, fetchFromGitHub, dbus, gdk_pixbuf, libnotify, makeWrapper, pkgconfig, xorg, alsaUtils }:
 
-let
-  runtimeDeps = [ xorg.xsetroot ]
-    ++ lib.optional (alsaUtils != null) alsaUtils;
-in
-
 rustPlatform.buildRustPackage rec {
   name = "dwm-status-${version}";
-  version = "0.4.0";
+  version = "0.5.0";
 
   src = fetchFromGitHub {
     owner = "Gerschtli";
     repo = "dwm-status";
     rev = version;
-    sha256 = "0nw0iz78mnrmgpc471yjv7yzsaf7346mwjp6hm5kbsdclvrdq9d7";
+    sha256 = "1vfajhdgvgsq6fvqmnq834incr4ijs6ihv3q8iaf26s1qiqhfl95";
   };
 
   nativeBuildInputs = [ makeWrapper pkgconfig ];
-  buildInputs = [ dbus gdk_pixbuf libnotify ];
+  buildInputs = [ dbus gdk_pixbuf libnotify xorg.libX11 ];
 
-  cargoSha256 = "0169k91pb7ipvi0m71cmkppp1klgp5ghampa7x0fxkyrvrf0dvqg";
+  cargoSha256 = "0qr999hwrqn7a4n4kvbrpli7shxp9jchj8csxzsw951qmzq32qwv";
 
-  postInstall = ''
+  # needed because alsaUtils is an optional runtime dependency
+  postInstall = lib.optionalString (alsaUtils != null) ''
     wrapProgram $out/bin/dwm-status \
-      --prefix "PATH" : "${stdenv.lib.makeBinPath runtimeDeps}"
+      --prefix "PATH" : "${alsaUtils}/bin"
```

Comment:
`xsetroot` is no longer needed?

Reply:
Yes, I set the root window name via libX11